File: pset1/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //makes sure user enters number between 0 and 23//

    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (0 > n || n > 23);

    //creates pyramid//

    for (int r = 0; r < n; r++)
    {
        for (int s = 0; s < n - 1 - r; s++)
        {
            printf(" ");
        }
        for (int b = 0; b < r + 2; b++)
        {
            printf("#");
        }

        printf("  ");

        for (int b = 0; b < r + 2; b++)
        {
            printf("#");
        }
        for (int s = 0; s < n - 1 - r; s++)
        {
            printf(" ");
        }
        printf("\n");
    }
}

File: pset6/mario/less/mario.py
import sys
from cs50 import get_int


def main():

    # prompt user for height
    while True:
        n = get_int("Height: ")
        if n > 0 and n < 9:
            break

    # build pyramid
    for i in range(n):
        for j in range(n - 1 - i):
            print(" ", end="")
        for k in range(i + 1):
            print("#", end="")
        print()


# calls main
if __name__ == "__main__":
    main()